bitcoin/chainparams.h

```diff
@@ -0,0 +1,34 @@
+#ifndef LIGHTNING_BITCOIN_CHAINPARAMS_H
+#define LIGHTNING_BITCOIN_CHAINPARAMS_H
+
+#include "config.h"
+#include <bitcoin/shadouble.h>
+#include <ccan/short_types/short_types.h>
+#include <stdbool.h>
+
+struct chainparams {
+	const int index;
+	const char *network_name;
+	const struct sha256_double genesis_blockhash;
+	const int rpc_port;
+	char *cli;
+	char *cli_args;
```

Comment:
The problem is that we are adding them to an array of non-const char pointers during the command line construction, and I could not find a way to shut the compiler up about dropping the const when adding to the array. What would the canonical way of doing that be?